stores/listingCart.ts

```diff
@@ -2,6 +2,7 @@ import { defineStore } from 'pinia'
 import { EntityWithId, NFTMetadata } from '@/components/rmrk/service/scheme'
 import { ComputedRef } from 'vue'
 import type { Prefix } from '@kodadot1/static'
+import { existInItemIndex } from '@/components/common/shoppingCart/utils'
```

Comment:
import from one store to another is not a good practice
if you want this wrapped as a function, it should be a getter in the stores

Reply:
You are right, we can refactor this